### PR TITLE
Add issue template config

### PR DESCRIPTION
**Pull request by MartinHjelmare**
fixes https://github.com/home-assistant-libs/zwave-js-server-python/issues/135

### CODE REVIEW

**Comment by MartinHjelmare** [comment]
@AlCalzone do you want me to say anything more or add other links?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Report a bug with the Z-Wave JS
```
Comment (AlCalzone) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
  - name: Report a bug with Z-Wave JS
```

Comment (AlCalzone) [reply .github/ISSUE_TEMPLATE/config.yml:3]: Otherwise okay

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (MartinHjelmare) [reply .github/ISSUE_TEMPLATE/config.yml]: Thanks!

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**